llvm/lib/Transforms/Instrumentation/MemorySanitizer.cpp

```diff
@@ -5920,11 +5995,6 @@ struct MemorySanitizerVisitor : public InstVisitor<MemorySanitizerVisitor> {
                                         /*trailingVerbatimArgs=*/1);
       break;
 
-    case Intrinsic::x86_avx512_mask_cvtps2dq_512: {
```

Comment:
you moved this around to group "convert" stuff, right?

Reply:
Yes, that was the intent. For simplicitly, I've undone the change. (I'll move it around in a future NFC patch.)